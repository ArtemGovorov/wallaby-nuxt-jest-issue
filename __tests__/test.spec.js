// Vue Testfile
import { mount } from '@vue/test-utils';
import TestComp from '@/components/testcomp.vue';

describe('Testing Testcomp', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(TestComp);
  });

  it('is vue instance', () => {
    expect(wrapper.name()).toBe('testcomp');
    expect(wrapper.isVueInstance()).toBeTruthy();
  });
});

