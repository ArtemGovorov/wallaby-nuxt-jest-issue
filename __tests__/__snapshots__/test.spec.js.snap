// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Testcomp Render correctly 1`] = `
<div>
  <!---->
  This is a test
</div>
`;
